content/docs/introduction/design-doc.md

```diff
@@ -3,51 +3,44 @@ title: Design Documents
-| [OpenMetrics transition](https://docs.google.com/document/d/1VuUBdRyIDR2uID2j2abWGt6PbWu7o-s06btZgrwH4vQ/edit?usp=sharing) | 2022-06-24 | In progress | |
-| Semantics of muting in Alertmanager | | TODO | |
-| [Extrapolation in range selectors (xrate)](https://docs.google.com/document/d/1y2Mp041_2v0blnKnZk7keCnJZICeK2YWUQuXH_m4DVc/edit#) | | Under review | |
-| Serverless, MQTT, and IoT use cases in the Prometheus ecosystem | | TODO | |
-| Static arithmetic for timestamps and durations | | TODO | |
-| [Prometheus long-term supported releases](https://docs.google.com/document/d/1wCXLnvh460UG10Mw019UbYMWDwSWWdvgwY61LHY0A0w/edit) | 2022-03-02 | Proposed | |
-| [Support for out of order samples in the TSDB](https://docs.google.com/document/d/1Kppm7qL9C-BJB1j6yb6-9ObG3AbdZnFUBYPNNWwDBYM/edit) | 2022-04-13 | Implemented | [#11075](https://github.com/prometheus/prometheus/pull/11075) |
-| [Structured Remote Write Protocol](https://docs.google.com/document/d/1i880vXorPec1ls91bnuoKR3-Hl3-D9XA1q1w734rZnU/edit#) | 2022-04-25 | proposed | |
+# Design Documents
+
+It's essential to clearly explain the reasons behind certain design decisions to have a community consensus. This is especially
+important in Prometheus, where every decision might have a significant impact given the high adoption and stability of the software and standards we work on.
+
+In our world, no decision is perfect, so having a design document that explains the trade-offs we made is essential.
+Such a document can also be used later on as a reference and for knowledge-sharing purposes.
+
+Note that design documents do not always reflect exactly what has been (or will be) implemented. Implementation details
+might have changed since a feature was merged. Design docs are not considered documentation and can not define a standard.
+Instead, it should explain the motivation, scope, the decision made and alternatives considered.
+
+## Current proposals
+
+The [github.com/prometheus/proposals](https://github.com/prometheus/proposals)repository holds all the past and current proposals for the Prometheus Ecosystem. It's the single place for reviewing, discovering and working on the design documents. It also a record of the past decisions and approvals.
+
+* [The PRs with proposal label](https://github.com/prometheus/proposals/pulls?q=is%3Aopen+is%3Apr+label%3Aproposal) shows all the pending proposals.
+* The [proposals directory](https://github.com/prometheus/proposals/tree/main/proposals) shows all the accepted proposals. See the "Implementation Status" for details on the implementation.
+* [The unmerged PRs with proposal label](https://github.com/prometheus/proposals/pulls?q=is%3Apr+label%3Aproposal+is%3Aclosed+is%3Aunmerged) shows all the rejected proposals.
+
+## Proposal process
+
+Don't get scared to propose ideas! It's amazing to innovate in the open and get feedback on the ideas.
+
+The process of proposing change with the design document would look as follows:
```

Comment:
Maybe "The process of proposing a change via a design document is the following:" ?